### ISSUE TITLE
Release notes old and no changelog

### ISSUE THREAD

**Issue by GrimLink** [argyleink/roving-ux#18]
Currently roving-ux is at v1.0.5,
but the release notes are only for v1.0.2 and older.

There is also no changelog to at least show any changes.

Consider adding a changelog or update the release notes to the current version

**Comment by argyleink** [comment]
good call,  i'm pretty bad at release notes and its def a place for improvement. thanks for the feedback 👍🏻
